rhel/SPECS/dbaas-controller.spec

```diff
@@ -0,0 +1,64 @@
+%undefine _missing_build_ids_terminate_build
+%global _dwz_low_mem_die_limit 0
+
+# do not strip debug symbols
+%global debug_package   %{nil}
+
+%global provider	github
+%global provider_tld	com
+%global project		percona-platform
+%global repo		dbaas-controller
+%global provider_prefix	%{provider}.%{provider_tld}/%{project}/%{repo}
+%global import_path	%{provider_prefix}
+%global commit		41d72afe43a88f522bdd35a4281a447424f18055
+%global shortcommit	%(c=%{commit}; echo ${c:0:7})
```

Comment:
I think we should start cleaning up that crazy stuff.

provider/provider_tld/etc should be just removed.

IIRC, `%global commit 41d72afe43a88f522bdd35a4281a447424f18055` is `sed`'ed during the build, right? So let's use some value that indicates that, like "see build script" or "0000000000000000000000000000000000000000" with a comment or something. Otherwise, it looks like we build fixed old commit.

Reply:
I support this

Reply:
![image](https://user-images.githubusercontent.com/11512/87540953-2625b700-c6a9-11ea-80d9-ad4abf8224d0.png)

Can we align those lines with spaces?

Reply:
> provider/provider_tld/etc should be just removed.

You also missed that part